tests/test_general_linear.py

```diff
@@ -127,8 +127,12 @@ def test_exp_vectorization(self):
                              [[2.718281828, 0., 0.],
                               [0., 148.413159, 0.],
                               [0., 0., 403.42879349]]])
+
+        expected = gs.cast(expected, gs.float64)
```

Comment:
🎉

Reply:
In some cases, casting float32 to float64 allows to reach the desires tolerance